pkg/minikube/registry/drvs/virtualbox/virtualbox.go

```diff
@@ -111,5 +111,15 @@ func status() registry.State {
 		}
 	}
 
-	return registry.State{Installed: true, Healthy: true}
+	version := ""
+	cmd = exec.CommandContext(ctx, path, "--version")
```

Comment:
how about we combine this command with the one above?
instead of runnig
```
	cmd := exec.CommandContext(ctx, path, "list", "hostinfo")
```
we could run the version
that we we save time by running one command instead of two and save the version to be used too

Reply:
Hi @medyagh, I made the requested changes, Please review